### PR TITLE
Bkearney/add i18n

### PR DESCRIPTION
**Pull request by bkearney**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,6 +2,8 @@ source "https://rubygems.org"
 
 gemspec
 
+gem 'gettext', '~> 2.0'
```
Comment (komidore64) [inline Gemfile]: add this to the gemspec as opposed to the Gemfile.

```ruby
spec.add_development_dependency 'gettext', '~> 2.0'
```

File: rel-eng/releasers.conf [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[git-sat]
+releaser = tito.release.DistGitReleaser
+branches = satellite-6.1.0-rhel-7 satellite-6.1.0-rhel-6
```
Comment (komidore64) [inline rel-eng/releasers.conf]: is it okay to check this stuff in downstream?

File: rubygem-hammer_cli_csv.spec [hunk 3]
```diff
@@ -1,34 +1,35 @@
...
 Summary: CSV input/output command plugin for the Hammer CLI
 Name: rubygem-%{gemname}
-Version: 0.0.6
+Version: 0.0.6.3
 Release: 1%{?dist}
 Group: Development/Languages
 License: GPLv3
 URL: https://github.com/Katello/hammer-cli-csv
-Source0: %{gemname}-%{version}.gem
+Source0: %{gemname}-%{gemversion}.gem
```
Comment (komidore64) [inline rubygem-hammer_cli_csv.spec]: why the change to `gemversion`? `gemversion` isn't defined above.

Comment (komidore64) [inline rubygem-hammer_cli_csv.spec]: it seems like a bad idea to have the gemversion differ from the RPM's version.

**Comment by bkearney** [comment]
I am gonna delete this and submit a new one from a new repo.
